dataland-user-service/src/test/kotlin/org/dataland/datalanduserservice/service/CompanyReportingInfoServiceTest.kt

```diff
@@ -49,11 +49,12 @@ class CompanyReportingInfoServiceTest {
         @JvmStatic
         fun paramStream(): Stream<Arguments> =
             Stream.of(
-                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2025, 12, 1), 2026),
-                Arguments.of(LocalDate.of(2024, 1, 31), -1, LocalDate.of(2025, 12, 1), 2025),
-                Arguments.of(LocalDate.of(2018, 1, 31), 0, LocalDate.of(2025, 12, 1), 2026),
-                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2021, 12, 1), 2022),
-                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2025, 6, 1), null),
+                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2025, 1, 16), null),
+                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2025, 5, 31), 2025),
+                Arguments.of(LocalDate.of(2018, 1, 31), -1, LocalDate.of(2025, 5, 31), 2024),
+                Arguments.of(LocalDate.of(2024, 12, 31), 0, LocalDate.of(2025, 5, 31), 2024),
+                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2025, 6, 1), 2025),
+                Arguments.of(LocalDate.of(2024, 12, 31), 0, LocalDate.of(2025, 7, 16), null),
```

Comment:
For these tests, I would have loved to see a similar structure as the previous tests. Previously, it was:
1) Standard case
2) Same case, but with different reporting period shift
3) Same case, but with a different year in the FYE
4) Same case, but FYE is now in the future
5) Different case where there the expectedReportingPeriod is null. 

An example could be
```suggestion
                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2025, 5, 31), 2025),
                Arguments.of(LocalDate.of(2024, 1, 31), -1, LocalDate.of(2025, 5, 31), 2024),
                Arguments.of(LocalDate.of(2018, 1, 31), 0, LocalDate.of(2025, 5, 31), 2025),
                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2021, 5, 31), 2021),
                Arguments.of(LocalDate.of(2024, 12, 31), 0, LocalDate.of(2025, 7, 16), null),
                Arguments.of(LocalDate.of(2024, 1, 31), 0, LocalDate.of(2025, 1, 16), null),
```